skorch/net.py

```diff
@@ -852,6 +854,27 @@ def fit(self, X, y=None, **fit_params):
         self.partial_fit(X, y, **fit_params)
         return self
 
+    def check_is_fitted(self, attributes=None, *args, **kwargs):
```

Comment:
Is `check_is_fitted` public to signal that it can be overridden by subclasses?

Reply:
Yes, the idea was to make it easy for a user to change the attributes that they consider to be necessary for a net to count as fitted.